From 19807916ba8576eadf91cf783b2474115ba733ac Mon Sep 17 00:00:00 2001
From: Alin Jerpelea <alin.jerpelea@sony.com>
Date: Tue, 30 Jan 2018 03:16:10 +0900
Subject: [PATCH] post_proc: Enable post processing for sdm660

the sdm660 SOC can use the same prost_proc code as all the other
QCOM SOCs

Test: manual - buils, boots, sound works

Change-Id: I18b9cec56c6197b4465e8009c7e50aa95e111d32
Signed-off-by: Alin Jerpelea <alin.jerpelea@sony.com>
---
 post_proc/Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/post_proc/Android.mk b/post_proc/Android.mk
index 2734d26..de31616 100644
--- a/post_proc/Android.mk
+++ b/post_proc/Android.mk
@@ -1,4 +1,4 @@
-ifneq ($(filter msm8974 msm8226 msm8084 msm8992 msm8994 msm8996 msm8909 msm8952 msm8998,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8974 msm8226 msm8084 msm8992 msm8994 msm8996 msm8909 msm8952 msm8998 sdm660,$(TARGET_BOARD_PLATFORM)),)
 
 LOCAL_PATH:= $(call my-dir)
 
@@ -43,7 +43,7 @@ endif
 
 ################################################################################
 
-ifneq ($(filter msm8992 msm8994 msm8996 msm8909 msm8952 msm8998,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8992 msm8994 msm8996 msm8909 msm8952 msm8998 sdm660,$(TARGET_BOARD_PLATFORM)),)
 
 include $(CLEAR_VARS)
 
-- 
2.15.1

